biz.aQute.bndlib/src/aQute/bnd/osgi/FileResource.java

```diff
@@ -51,7 +52,7 @@ public String toString() {
 	}
 
 	public void write(OutputStream out) throws Exception {
-		copy(this, out);
+		Files.copy(file.toPath(), out);
```

Comment:
Given the choice, if both the JCL and an external library provided a method that performed the same function, I would always choose the JCL provided method. It may have native bindings under the hood. It may be able to make better decisions about block sizes, etc.

While profiling a build, I noticed we spent a measurable amount of time in FileResource.copy(), so I assumed this was (at worst) a zero sum change.